docs/source/get_started/distributed_training.rst

```diff
@@ -269,7 +269,7 @@ You can create a `yaml` file called `test.yaml` and set all the config as follow
         group_by: user
         order: RO
         split: {'RS': [0.8,0.1,0.1]}
-        mode: full
+        mode: {"valid": "full", "test": "full"}
```

Comment:
Hello, @ChenglongMa. What wonderful code! Thanks very much for your attention to RecBole!

As for the `mode` in `eval_args`, maybe we can keep the original setting, but the interface of the dictionary is extended.  In other words, we still allow users to input `full` or `uni100` as the mode. In this case, the evaluation and test modes are the same. At the same time, we also provide a dictionary interface. If the user inputs the form of the dictionary, we judge whether there are `valid` and `test` key values, and set the corresponding evaluation mode. That is to say, only when different modes are needed for evaluation and testing, the dictionary form is needed to facilitate compatibility with previous configurations.

We would appreciate it if it would be convenient for you to make further modifications. Thank you again for your contributions to RecBole!